ndscan/experiment/fragment.py

```diff
@@ -316,25 +316,30 @@ def setattr_param(self, name: str, param_class: Type, description: str, *args,
 
     def setattr_param_rebind(self,
                              name: str,
-                             original_owner: "Fragment",
-                             original_name: Union[str, None] = None,
+                             original_handle: ParamHandle,
+                             additional_handles: Optional[List[ParamHandle]] = None,
```

Comment:
We could consider splatting (`*additional_handles`). That way, it is explicit in the function declaration which handle is used to create the new parameter, but it also represents the idea of "rebind one or more parameters" in the way the function is called, eg `setattr_param_rebind("param", param0, param1, param2)`. What do you think?

Reply:
hmmm...I don't have a strong feeling here but to me that doesn't really achieve the aim of making it 100% explicit which parameter is the template so I (weakly) prefer the signature currently in this PR.